schema/draft/schema.ts

```diff
@@ -1136,11 +1141,11 @@ export interface CompleteRequest extends Request {
      * Additional, optional context for completions
      */
     context?: {
-       /**
-        * Previously-resolved variables in a URI template or prompt.
-        */
-        arguments?: { [key: string]: string };
-     };
+      /**
```

Comment:
Sorry, my IDE is fixing formatting in unrelated places. But I think we should keep them.